### ISSUE TITLE
Section and Tab Translation is not working

### ISSUE THREAD

**Issue by ManuelBuecherl** [MscrmTools/MsCrmTools.Translator#47]
Hi there, 

since a few weeks we face an issue with Form Translation.
When importing Form **Tab** or Form **Section** Translations, they are getting imported successfully according to the Status of the Easy Translator, but they are not visible in the System.

When we then export the standard Translation ZIP, I can see, that the Translation did not reach the System.

Is that Issue already known?

Best Regards, 
Manuel

---
- Deployment: Online
- DB Version: 9.2.21064.144
- Connection Controls Version: 1.2021.6.44
- XrmToolBox Version: 1.2021.6.51
- Tool Version: 1.2021.4.15

**Comment by MscrmTools** [comment]
Hi, this is not a known issue.
Maybe something has changed on Microsoft side. I will try to investigate but I do not have lot of time these days

**Comment by LarsBauer** [comment]
Unfortunately this issue still occurs. I am unable to update the translation of a section I newly added to a form. The default translation export/import is working fine. Any update on this? Is there anything we can support you with?

**Comment by ingerAH-PP** [comment]
We are facing the same issue, translation of form tabs and sections are not updated in the system.

Specs:

- Deployment: Online

- DB Version: 9.2.22012.00138

- Connection Controls Version: 1.2021.9.46

- XrmToolBox Version: 1.2021.12.53

- Easy Translator Version: 1.2022.2.17

**Comment by MscrmTools** [comment]
@ingerAH-PP @LarsBauer I'm working on this issue currently. Can you tell me if the issue occurs for the base language of your organization or even for other languages?

**Comment by LarsBauer** [comment]
Thanks for the update. My base language is set to English (1033) and I am unable to translate sections to German (1031).

**Comment by MscrmTools** [comment]
ok, thank you. I will continue to investigate and let you know the result

**Comment by MscrmTools** [comment]
As per my test, even using standard translation process does not translate tabs and sections. I've opened a support request and I'm talking with the product team to understant why it does not work as expected

**Comment by MscrmTools** [comment]
@LarsBauer @ingerAH-PP Would you be kind to test this version? I do succeed to translate tabs and sections with it
[MsCrmTools.Translator.zip](https://github.com/MscrmTools/MsCrmTools.Translator/files/8117280/MsCrmTools.Translator.zip)

**Comment by MscrmTools** [comment]
@LarsBauer @ingerAH-PP , would you have time to test the version I sent in the previous post? It would help close the case with Microsoft.

**Comment by ingerAH-PP** [comment]
I have tested the version you sent (1.2022.2.19). I still experience the same issue, neither tabs and sections are being translated. Form fields are successfully translated.

**Comment by MscrmTools** [comment]
That is strange... did you hard refresh your browser (Ctrl+F5) before checking the result?

**Comment by ingerAH-PP** [comment]
Yes, and I have tried exporting translations for the same table that was recently translated and imported. The export shows that the translations for the form tabs and sections are empty. So the translations that were recently imported have somehow not been published.

**Comment by gvbooost** [comment]
Any news on this matter? I experience the same behaviour (base: English, additional Dutch).

I have had trouble before half a year a go, maybe related or otherwise helping to find out: base language english, additional language dutch. For standard Dynamics objects the Dutch labels had been overwritten overnight by English ones. They had to remove the unmanaged solution layer for all "infected" objects one-by-one using api calls. This all happened after language pack updates pushed by Microsoft. :(

**Comment by MscrmTools** [comment]
No progress so far but thank you for mentioning unmanaged layer. This is a path I need to check

**Comment by gvbooost** [comment]
Just learned that exporting from a solution in the PowerApps maker app (instead of classic), fortunately does the job, although it is the same terrible spreadsheet format as before. Besides that, the new command button labels are exported too. So they have been working on it, as it seems. 
Hope you can find the cause and fix it, as your tool helps to find and update what you really need to, without getting lost. Many thanks in advance.

**Comment by MscrmTools** [comment]
I think I finally succeeded to fix this issue! A new version is coming in the next hours or so. Please test it to validate it works as expected

**Comment by fscattolin** [comment]
This issue is still present on latest version. 

I discovered the following behavior.
You cannot traslante tabs ad section created using the new form designer. With Tabs and section created using legacy form editor Easy Translator works as expected.

**Comment by zonton** [comment]
Issue is still there for me as well. 
I tried many different setup to import fewer records without success.
I also tried to create a new main form in the legacy designer as fscattolin mentioned but the translations are not working either. (New Main Form then Save as)
I'm translating from French (Base Language) to English.

Keep up the great work, many thanks in advance

**Comment by zonton** [comment]
Any news ? 
Thank you very much.

**Comment by MscrmTools** [comment]
Hi @zonton, @fscattolin, @ManuelBuecherl , @ingerAH-PP , @arlassen 

I'm using latest version of this tool and I succeeded to translate tabs, sections and forms. See video below where I replace the suffixes (FR) and (US) of labels by (FR2) and (US2).
https://www.screencast.com/t/C6V0p2m9Lx

What I noticed is that labels are cached so that you need to Ctrl+F5 to see the change but it does work

My base language is French and the additional language is English (United States)
